lib/router.js

```diff
@@ -6,43 +6,43 @@ const mixins = hof.mixins;
 const i18nFuture = hof.i18n;
 const router = require('express').Router();
 const path = require('path');
-const debug = require('debug')('router');
 
-module.exports = (routeConfig, config) => {
-  const baseUrl = routeConfig.baseUrl || '/';
-  debug('create %s', baseUrl);
+module.exports = (config) => {
 
-  let commonFields;
-  let fields = require(path.resolve(config.caller, routeConfig.fields));
-  const translations = path.resolve(config.caller, routeConfig.translations ? path.resolve(routeConfig.translations, config.translations) : config.translations)
+  const baseUrl = config.route.baseUrl || '/';
+  const routeFields = path.resolve(config.caller, config.route.fields || config.fields);
 
-  if (routeConfig.commonFields) {
-    commonFields = require(path.resolve(config.caller, routeConfig.commonFields));
+  let fields;
+
+  try {
+    fields = require(path.resolve(config.caller, config.fields));
+  } catch (err) {
+    fields = err;
   }
 
-  if (commonFields && fields) {
-    fields = Object.assign({}, commonFields, fields);
+  if ((fields instanceof Error) === false) {
+    fields = Object.assign({}, fields, routeFields);
   }
```

Comment:
The `if` expression already evaluates to a `boolean`. It can be replaced with `if(!(fields instanceof Error))`